src/EpisodeManagementService/GetEpisode/Program.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NHS.ServiceInsights.Common;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
+    .ConfigureServices(services =>
+    {
+        services.AddHttpClient();
```

Comment:
Replace this with configuring DI for HttpRequestService

Reply:
Done